src/kernel/kvm_platform.c

```diff
@@ -46,6 +46,8 @@ static boolean probe_kvm_pvclock(kernel_heaps kh)
     }
     kvm_debug("pvclock detected");
     struct pvclock_vcpu_time_info * vc = allocate(backed, backed->pagesize);
+    if (vc == INVALID_ADDRESS)
```

Comment:
This makes the result a little ambiguous (no pvclock vs alloc fail) and a failure to get a page during platform init is pretty bad - maybe an assert is better here?